src/main/java/com/EndavaTicketManagement/practica/service/modelDTO/EventDto.java

```diff
@@ -0,0 +1,19 @@
+package com.EndavaTicketManagement.practica.service.modelDTO;
+
+public class EventDto {
+
+    private int ID;
+
+    public int getID() {
+        return ID;
```

Comment:
Te rog sa adaugi aici mai multe detalii sau detaliile de care ai nevoie pentru UI, in rest arata bine

Reply:
La fel si la OrderDTO